docs/migration.rst

```diff
@@ -0,0 +1,63 @@
+Migrating from python-fedora
+----------------------------
+
+Fasjson-client provides functionality for the most important endpoints previously exposed in python-fedora.
+Below is a list of common python-fedora endpoints and their fasjson-client alternatives.
+
+
+Creating a client
+******************
+
+The instantiation of a client can be done similarly to python-fedora, except there is no need to provide a username and password with which to authenticate. Instead, this authentication is performed by your service via Kerberos.
+
+For more information, please see :ref:`usage-label`. The content below assumes you have setup your client as detailed in :ref:`usage-label`.
+
+Pagination is supported in some of the `fasjson_client` API calls listed below, for more information about how to use it see :ref:`pagination-label`.
```

Comment:
In RST, preformatted inline text is done with two backticks before, two backticks after. If you only put one it's going to be read as a reference, so it will show up differently, but not as you want to.

Reply:
K thanks, I've left these as links and changed the others to double backticks.